app/src/main/java/com/example/sopt_assignment_dabin/Follower/FollowerRecyclerViewAdapter.kt

```diff
@@ -0,0 +1,49 @@
+package com.example.sopt_assignment_dabin.Follower
+
+import android.content.Intent
+import android.icu.text.IDNA
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.core.content.ContextCompat
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import com.example.sopt_assignment_dabin.R
+import com.example.sopt_assignment_dabin.databinding.ItemFollowerListBinding
+
+class FollowerRecyclerViewAdapter : RecyclerView.Adapter<FollowerRecyclerViewAdapter.FollowerViewHolder>() {
+    val followerList = mutableListOf<FollowerListData>()
+
+    inner class FollowerViewHolder(private val view: ItemFollowerListBinding) : RecyclerView.ViewHolder(view.root) {
+        fun onBind(data: FollowerListData) {
+            view.tvName.text = data.name
+            view.tvStory.text = data.story
+            Glide.with(itemView.context).load(data.image).circleCrop().into(view.ivFollowerImage)
+
+            itemView.setOnClickListener {
+                val intent=Intent(itemView?.context,DetailActivity::class.java).apply{
+                    putExtra("name", data.name)
+                    putExtra("image", data.image)}
+                ContextCompat.startActivity(itemView.context,intent,null)
+            }
+        }
+    }
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): FollowerRecyclerViewAdapter.FollowerViewHolder {
+        val view = ItemFollowerListBinding.inflate(LayoutInflater.from(parent.context), parent, false)
+        return FollowerViewHolder(view)
```

Comment:
되도록이면 inline으로 작성하시는게 가독성에 좀 더 좋을 것 같네용

```suggestion
    return FollowerViewHolder(ItemFollowerListBinding.inflate(LayoutInflater.from(parent.context), parent, false))
```